.github/ISSUE_TEMPLATE/version_release.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Version Release
+about: Checklist to follow when shipping a new release.
+title: 'Release <V1.x.x> Checklist'
+labels: ''
+assignees: ''
+
+---
+
+<!-- markdownlint-disable MD034 -->
+- [ ] Complete [Milestone](https://github.com/open-telemetry/opentelemetry-go/milestone/<Release Milesone>)
```

Comment:
It's maybe because I'm not a maintainer, but I can't figure out a way to get a working URL for this v1.8.0 milestone, outside of the issues interface..